mindsdb/utilities/log.py

```diff
@@ -41,6 +41,11 @@ def write(self, message):
     def flush(self):
         pass
 
+    def isatty(self):
```

Comment:
Added for coverage of methods that the class this replaces has, because other libraries (e.g. Ray) use them.